src/liapunovspectrum.jl

```diff
@@ -171,7 +173,7 @@ function liap_taylorinteg(f, q0::Array{U,1}, t0::T, tmax::T,
     const jt = eye(U, dof)
 
     # NOTE: This changes GLOBALLY internal parameters of TaylorN
-    global _δv = set_variables("δ", order=1, numvars=dof)
+    global _δv = set_variables(Taylor1{U}, "δ", order=1, numvars=dof)
```

Comment:
We should add a clause in the docstrings about this global change.

Reply:
Thinking it over, maybe we should rewrite this checking if the current internal parameters of TaylorN allow to skip this redefinition, so we define `_δv` is simply the "order one" independent variables. (Maybe the user cares about having a larger order.) To do this, we should check that the number of variables is compatible. This is only a proposal, which may be thrown away if it does not improve performance and allocations.

Reply:
Thanks for the suggestion! I'll try it out!

Reply:
I also was thinking of adding a warning whenever the `_δv` are set inside `liap_taylorinteg`? What do you think?